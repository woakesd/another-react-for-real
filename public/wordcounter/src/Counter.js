function Counter({ count }){
    return (
        <p className="mb2">
            Word count {count}
        </p>
    );
}

