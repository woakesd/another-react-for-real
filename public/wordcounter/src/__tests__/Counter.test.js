import React from 'react';
import renderer from 'react-test-renderer';
import Counter from '../Counter';

describe('a counter', () => {
    it('describes the count and label', () => {
        const counter = renderer.create(<Counter legend="Count" count={34} />);
        expect(counter.toJSON()).toMatchSnapshot();
    });
});