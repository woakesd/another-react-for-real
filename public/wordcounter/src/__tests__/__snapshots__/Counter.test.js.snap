// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`a counter describes the count and label 1`] = `
<p
  className="mb2"
>
  Word count 
  34
</p>
`;
